README.md

```diff
@@ -96,8 +96,9 @@ Finch has a simple and extensible configuration. A configuration file at `${HOME
 
 Currently, the options are:
 
-* CPUs [int]: the amount of vCPU to dedicate to the virtual machine
-* Memory [string]: the amount of memory to dedicate to the virtual machine
+* CPUs [int] (required): the amount of vCPU to dedicate to the virtual machine
+* Memory [string] (required): the amount of memory to dedicate to the virtual machine
+* AdditionalDirectories [AdditionalDirectory{path [string]}] (optional): the work directories that are not supported by default. In macOS, only home directory is supported by default.
```

Comment:
Maybe we can follow [what lima is doing](https://github.com/lima-vm/lima/blob/master/examples/default.yaml) to put descriptions and example values of the fields together in a single `.yaml` file and provide an [entry point in README](https://github.com/lima-vm/lima#configuration):

- For each field, reading the description of it together with an example value may make it easier to follow.
- `[AdditionalDirectory{path [string]}]` implies that we may need to add a new field here whenever a new field is added to the `AdditionalDirectory` struct, which can be a bit hard to read in the end, and maybe directly looking at the example values is easier to comprehend the structure of it:

```
additional_directories:
  - path: /Volumes
```

Reply:
The README already has an example yaml inline. Having a separated yaml file is a good idea. Current config is still very simple with 3 fields so I think inline yaml should be sufficient. Maybe we can have a separated yaml when there are more fields coming.

Reply:
> The README already has an example yaml inline. 

Yes, and I'm suggesting to merge that thing and the descriptions here into a single YAML file as I feel that separating them may decrease the readability according to the reasons stated above. Types like `[AdditionalDirectory{path [string]}]` don't really add much value, but we feel that we should provide it because currently the description and the example value are separated.